minio/Minio-connector.ipynb

```diff
@@ -0,0 +1,176 @@
+{
```

Comment:
Please, do not include this file in this PR as it belongs to a different topic/feature